datastore/dbkeys.go

```diff
@@ -38,6 +38,12 @@ const (
 	// Key used to store the auth token.
 	AuthTokenName = "authtoken"
 
+	// Key used to store the auth token.
+	AuthTenantTokenName = "tenanttoken"
+
+	// Key used to store the auth token.
+	AuthServerURLName = "serverurl"
```

Comment:
Since these are related to the auth token, I think their values should reflect that too. I suggest `authtoken_tenanttoken` and `authtoken_serverurl`. A bit verbose, but these database values are often long lived, so I think it's good to be explicit.

Reply:
thanks for all the comments&pointers will do!